src/handlers/meta_transaction_handlers.ts

```diff
@@ -365,10 +365,14 @@ const parseGetTransactionRequestParams = (req: express.Request): GetTransactionR
             },
         ]);
     }
-    const excludedSources =
+    const _excludedSources =
         req.query.excludedSources === undefined
             ? undefined
             : parseUtils.parseStringArrForERC20BridgeSources((req.query.excludedSources as string).split(','));
+    // Exclude Bancor as a source unless swap involves BNT token
+    const isBNT = sellToken.toUpperCase() === 'BNT' || buyToken.toUpperCase() === 'BNT';
+    const excludedSources = isBNT ? _excludedSources : _excludedSources.concat(ERC20BridgeSource.Bancor);
```

Comment:
Would this still work if `sellToken` or `buyToken` is provided as an address in the request?

Reply:
good catch, changed it to filter with both